src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Import/ImportResourceLoaderTests.cs

```diff
@@ -136,6 +136,54 @@ await foreach (ImportResource resource in outputChannel.Reader.ReadAllAsync())
             Assert.Equal(1, errorCount);
         }
 
+        [Fact]
+        public async Task GivenResourceLoader_WhenLoadResourcesWithSearchParameterResourceType_ThenResourcesWithSearchParameterTypeShouldBeSkipped()
```

Comment:
Given the suggested change in ImportResourceLoader, perhaps you can mock `ImportResourseParser` in the test like the `VerifyResourceLoaderAsync` method does to return a SearchParameter resource